scrapers/Pornbcn.yml

```
@@ -0,0 +1,34 @@
+name: PornBCN
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pornbcn.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //img[@data-name='image']/@alt
```

Found that this is often the Spanish title even if you're scraping an English link, suggest we replace this with a different selector to capture what the user sees
```suggestion
      Title: (//div[contains(@class, "rowdesktop1nsd")]//div[contains(@class, "et_pb_text_inner")])[1]
```